Gemfile

```diff
@@ -5,6 +5,15 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in rialto-etl.gemspec
 gemspec
 
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
+  "https://github.com/#{repo_name}.git"
```

Comment:
Can you add a comment atop this block saying that we're doing this because bundler + git protocol security, yada yada?